src/mocks.ts

```diff
@@ -0,0 +1,13 @@
+import consola from 'consola'
+
+export function mockConsola (): typeof consola {
+  const mock = {}
+
+  consola.mockTypes((type) => {
+    mock[type] = mock[type] || jest.fn()
```

Comment:
/cc @ricardogobbosouza. Consola does inherit mocked logs but here we should remember first mocked version otherwise after creating tag, a new mock (jest.fn()) will be created